src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+@import "../utils/variables.scss";
+
+.product-card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 48px;
+  }
+
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title-text {
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    line-height: 18px;
+    color: $dark;
+  }
+
+  &__code-product {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $gray;
+  }
+
+  &__raiting {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 24px;
+
+    &-comments {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 18px;
+      color: $dark;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-card {
+      color: #616070;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+
+    &-card_price {
+      color: $dark;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 14px;
+    }
+  }
+
+  &__btn {
+    display: inline-block;
+    height: 40px;
+    width: 100%;
+    padding: 12px 0;
+    font-weight: 700;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    color: #fff;
+    background-color: $blue;
+    border-radius: 5px;
+    line-height: 16px;
+    border: 1px solid $blue;
+  }
+
+  &__btn:hover {
+    background-color: #fff;
+    color: $blue;
+    transition: 0.5s;
```

Comment:
Transition should be written in &__btn selector, not in hover. This way it will work both when transition starts and when it ends. 
Also indicate which property you use transition for